src/qonlinetts.cpp

```diff
@@ -96,7 +96,8 @@ void QOnlineTts::generateUrls(const QString &text, QOnlineTranslator::Engine eng
 #if defined(Q_OS_LINUX)
             apiUrl.setQuery(query);
 #elif defined(Q_OS_WIN)
-            apiUrl.setQuery(query, QUrl::DecodedMode);
+            //apiUrl.setQuery(query, QUrl::DecodedMode);
+            apiUrl.setQuery(query); // fixed: Sometimes TTS engine pronounces space as "percent twenty"
 #endif
```

Comment:
Could you please replace this whole `#if...#endif` block with a simple `apiUrl.setQuery(query);`?

Reply:
also forgot to change it for yandex